hypha/static_src/src/sass/apply/custom/_custom.scss

```diff
@@ -1 +1,10 @@
 // stylelint-disable no-empty-source
+// stylelint-disable selector-class-pattern
+
+.list__item--average_score {
+    color: $color--dark-grey;
+    font-weight: $weight--normal;
+    font-size: map-get($font-sizes, zeta);
+    display: inline;
+    padding-left: 2px;
+}
```

Comment:
`_custom.scss` should be empty in stock hypha. It is used to add your custom css without worrying about merge conflicts.